### ISSUE TITLE
Integrate HTTPS data

### ISSUE THREAD

**Issue by gbinal** [GSA/site-scanning#890]
- [Example data here](https://docs.google.com/spreadsheets/d/1-fT3-5xt7PD7XMLNOtUzSWcA-v8NpaEuih5ASk4434o/edit#gid=1267952890)
- [Trimmed file hosted in GitHub](https://github.com/GSA/federal-website-index/blob/main/data/dataset/cisa_https.csv)

Gray/Lauren tasks needed to unblock Andy:

- ~Host sample file in Github and share location~ - done, [here](https://github.com/GSA/federal-website-index/blob/main/data/dataset/cisa_https.csv)
- Confirm field_ids, locations 
  - `domain` should be matched against Target URL
  - `domain_enforces_https` should be a new field known as `https_enforced`, located in the CSV after required_links_text
  - `hsts_base_domain_preloaded` should be a new field known as `hsts_preloaded`, located next, after `https_enforced`

Gray/Lauren tasks that aren't blockers: 
- Get real, live production data overwriting in GitHub
- Confirm which fields are included in GitHub file 
- Update documentation 
- More I'm sure ...

**Comment by gbinal** [comment]
@akuny - data is now uploaded [here](https://github.com/GSA/federal-website-index/blob/main/data/dataset/cisa_https.csv).

**Comment by gbinal** [comment]
Great work!  

Have reviewed this and we're a go to go ahead and add them to the snapshots and API.  FYI, I'll make another issue for this, but there's 116 entries that

**Comment by akuny** [comment]
These fields are now live in the snapshot and API.

**Comment by gbinal** [comment]
great work again, @akuny 

I've got more work to do via #896 but we'll close this one.
